index.bs

```diff
@@ -1524,17 +1532,36 @@ When this method is invoked, the [=authenticator=] must perform the following pr
     <figcaption>Generating an [=assertion signature=].</figcaption>
 </figure>
 
-On successful completion, the authenticator returns to the user agent:
+On successful completion, the [=authenticator=] returns to the user agent:
 - The identifier of the credential (credential ID) used to generate the [=assertion signature=].
 - The [=authenticator data=] used to generate the [=assertion signature=].
 - The [=assertion signature=].
 
-If the authenticator cannot find any credential corresponding to the specified [=[RP]=] that matches the specified criteria, it
+If the [=authenticator=] cannot find any credential corresponding to the specified [=[RP]=] that matches the specified criteria, it
 terminates the operation and returns an error.
 
 If the user refuses consent, the authenticator returns an appropriate error status to the client.
+    </dd>
+    <dt>"public-key-uaf"</dt>
+    <dd>
+- Follow the procedure as specified in [[FIDO-UAF-Authenticator-Commands]] section 6.3. Using
```

Comment:
Please look into step 7 (in UAF Authenticator Command spec), it specifies that 7.1 will only happen "If number of remaining RawKeyHandles is > 1".  This cannot be the case once the user has selected the preferred credential (note that the credential rawID translates into KeyHandle which is uwrapped/mapped into RawKeyHandle in step #4 in UAF Authenticator command).
So we are NOT running into a second credential selection process - we are just leveraging the fact the in the Web Authentication world the platform asks the user to disambiguate the credential.
So I think we are fine here.